src/renderer/containers/QuickStart/ImportDolphinSettingsStep.tsx

```diff
@@ -24,21 +27,32 @@ type FormValues = {
 
 export const ImportDolphinSettingsStep: React.FC = () => {
   const setExists = useDesktopApp((store) => store.setExists);
+  const desktopAppPath = useDesktopApp((store) => store.path);
   const { addToast } = useToasts();
 
   const migrateDolphin = async (values: FormValues) => {
-    await ipc_migrateDolphin.renderer!.trigger({
-      migrateNetplay: values.shouldImportNetplay ? values.netplayPath : null,
-      migratePlayback: values.shouldImportPlayback,
-    });
-    setExists(false);
+    if (values.shouldImportNetplay) {
+      await ipc_importDolphinSettings.renderer!.trigger({
+        toImportDolphinPath: values.netplayPath,
+        type: DolphinLaunchType.NETPLAY,
+      });
+    }
+    if (values.shouldImportPlayback) {
+      const binaryName = isMac ? "Slippi Dolphin.app" : "Slippi Dolphin.exe";
+      const playbackPath = path.join(desktopAppPath, "dolphin", binaryName);
```

Comment:
What are your thoughts on having `main` should just send `renderer` the path with the `Dolphin.exe` or `Dolphin.app` so `renderer` doesn't have to do this `path.join` stuff?

Reply:
yeah makes sense, we can go that route